Jellyfin.Plugin.Tvdb/TvdbClientManager.cs

```diff
@@ -48,6 +47,10 @@ public TvdbClientManager(IApplicationHost applicationHost)
 
     private static string? UserPin => TvdbPlugin.Instance?.Configuration.ApiKey;
 
+    private static int? CacheDurationInHours => TvdbPlugin.Instance?.Configuration.CacheDurationInHours;
+
+    private static int? CacheDurationInDays => TvdbPlugin.Instance?.Configuration.CacheDurationInDays;
```

Comment:
Just make these no nullable and clean up the rest of the code

Reply:
Changed